apps/react-example-counter/src/app/Counter.tsx

```diff
@@ -1,12 +1,17 @@
-import React from "react";
-import { useSubscription, useStore } from "@rx-store/react-rx-store";
-import {rootContext} from "./Manager";
+import React, { useMemo } from 'react';
+import { useSubscription, useStore } from '@rx-store/react-rx-store';
+import { rootContext } from './Manager';
+import { scanSum } from './store/operators/scan-sum';
 
 function Counter() {
   const store = useStore(rootContext);
 
   const [count] = useSubscription(store.count$);
-  const [localCount] = useSubscription(store.localCount$);
+
+  const localCount$ = useMemo(() => store.counterChange$.pipe(scanSum()), [
```

Comment:
@kylecannon  here is where I put localCount in react, on the consuming component, like an instance member on a React class component (its not reconciled as state since it never changes anyway). The useMemo is so it doesn't change, causing useSubscription to switchMap every render

Reply:
Addressed in 1c8bae8 to allow the example to work again. I would like to make an angular library to incorporate the changes cause I am not happy with the current way it looks and feels. I'll work on a followup PR or commit.